libs/template/application/src/push-notification/bapi/feature.ts

```diff
@@ -0,0 +1,24 @@
+import { AppFeature, injectEnv } from '@spryker-oryx/core';
+import {
+  PushNotificationFeature,
+  PushServiceConfig,
+} from '@spryker-oryx/push-notification';
+
+export class BapiPushNotificationFeature
```

Comment:
We do not need this feature at all.
Simply integrate PushNotificationsFeature in the FulfillmentFeature.